lms/product/d2l/product.py

```diff
@@ -8,3 +8,5 @@ class D2L(Product):
     """A product for D2L specific settings and tweaks."""
 
     family: Product.Family = Product.Family.D2L
+
+    settings_key = "desire2learn"
```

Comment:
This is the string we have not for the family, we have to do the mapping anyway so we could go for "d2l" instead. 

We are not typing this very often in any case.